pkg/api/customization/cluster/formatter.go

```diff
@@ -47,7 +47,23 @@ func (f *Formatter) Formatter(request *types.APIContext, resource *types.RawReso
 			resource.AddAction(request, v3.ClusterActionBackupEtcd)
 			resource.AddAction(request, v3.ClusterActionRestoreFromEtcdBackup)
 		}
-		resource.AddAction(request, v3.ClusterActionRunSecurityScan)
+		canRunScan := func() bool {
+			revision := map[string]interface{}{
+				"id": request.ID,
+			}
```

Comment:
Is this the Cluster's Id? maybe just checking if "update" on cluster resource is allowed should suffice here

Reply:
modified the logic, we don't need the id.